packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -55,17 +72,19 @@ export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
       </div>
 
       {filteredOptions.map(option => (
-        <button
-          className={classnames(
-            option === selectedOption && styles.selectedOption,
-          )}
-          role="button"
-          tabIndex={0}
-          key={option}
-          onClick={handleSelection}
-        >
+        <label htmlFor={option} key={option}>
```

Comment:
I don't think that it makes sense to keep these as buttons. Semantically a radio makes more sense here because a selection is taking place. 
In terms of the implementation for multiselect I was thinking that we would have a prop that would set a variable to dictate the type for the input and either make it radio or checkbox. that way the logic could stay relatively simple.

Reply:
just an fyi, radio's will automatically move the selected option to the next or previous using the arrow keys. Not sure if we want that. Do we want that? That seems different than other menu-type UI where you are finding an option before selecting it (HTML select, autocomplete).

Reply:
interesting, this radio button setup we have currently is doing something unexpected where I can't give focus via tabs to the radios at all

I can't tell if I'm being dumb, but I cannot for the life of me get the focus to move to the radios and using the arrows only ever scrolls either the inner list, or the outer Storybook page. it never changes between the radios/labels

Reply:
oh right. that involves some fun CSS trickery. https://atlantis.getjobber.com/?path=/story/components-selections-chips-web--basic

Since you don't focus on the label, you focus on the input radio